pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -220,10 +220,10 @@ func (rc *reconciler) mountOrAttachVolumes() {
 		volMounted, devicePath, err := rc.actualStateOfWorld.PodExistsInVolume(volumeToMount.PodName, volumeToMount.VolumeName, volumeToMount.PersistentVolumeSize, volumeToMount.SELinuxLabel)
 		volumeToMount.DevicePath = devicePath
 		if cache.IsSELinuxMountMismatchError(err) {
-			// TODO: check error message + lower frequency, this can be noisy
-			klog.ErrorS(err, volumeToMount.GenerateErrorDetailed("mount precondition failed, please report this error in https://github.com/kubernetes/enhancements/issues/1710, together with full Pod yaml file", err).Error(), "pod", klog.KObj(volumeToMount.Pod))
-			// TODO: report error better, this may be too noisy
-			rc.desiredStateOfWorld.AddErrorToPod(volumeToMount.PodName, err.Error())
+			// The volume is mounted, but with an unexpected SELinux context.
```

Comment:
Do you think a metric would be useful?

Reply:
No, a metric would  grow like crazy, this is called every 100ms. In addition, it is a part of the "good" workflow - similar to `IsVolumeNotAttachedError` below. Kubelet must unmount the old volume (with wrong SELinux label) first.

Reply:
Now that I think about it, calling `AddErrorToPod` still could be useful, in case unmount of the other volume takes too long.

Reply:
Added a separate commit with `AddErrorToPod`.

Regarding the metric - there already is a metric when a second pod with SELinux label that does not match the already present one is rejected and that's an error. To get a metric when a volume is being unmounted because of unexpected SELinux label, I would need to add something into `unmountDetachDevices`...